docs/path.md

```diff
@@ -2,6 +2,22 @@
 
 Since there does not exist a standard for path syntax, RedisJSON implements its own. RedisJSON's syntax is a subset of common best practices and resembles [JSONPath](http://goessner.net/articles/JsonPath/) not by accident.
 
+There is currently two concurrent implementations. One is a legacy from the first version of RedisJSON. Describe below as the `legacy path syntax`.
```

Comment:
What does an implementation "from the first version of RedisJSON" mean? That we still have the exact same implementation?
2.0 behavior is not identical to 1.0, although tries to have as much as backward compatibility as possible (we changed ARRTRIM negative start index behavior, some error messages, and maybe a few more differences)

Reply:
Rephrasing ...